src/Settings/components/reminders/ReminderOptions/hooks/useHeader.jsx

```diff
@@ -37,21 +40,19 @@ const useHeader = (baniListData, navigation, selector, setStateData) => {
     );
   };
   useEffect(() => {
-    if (baniListData.length > 0) {
-      navigation.setOptions({
-        title: constant.REMINDER_OPTIONS,
-        headerTitleStyle: {
-          color: colors.WHITE_COLOR,
-          fontWeight: "normal",
-          fontSize: 18,
-        },
-        headerStyle: {
-          backgroundColor: colors.TOOLBAR_COLOR_ALT2,
-        },
-        headerLeft,
-        headerRight: () => headerRight(baniListData),
-      });
-    }
+    navigation.setOptions({
+      title: STRINGS.set_reminder_options,
+      headerTitleStyle: {
+        color: colors.WHITE_COLOR,
+        fontWeight: "normal",
+        fontSize: 18,
```

Comment:
Is 18 here is like the default font size? If so, should it come from a constant variable, instead of hardcoded value?

Reply:
Default font is not yet set veer ji. However, I'll set in coming PR.